WordPress/Sniffs/WP/I18nSniff.php

```diff
@@ -0,0 +1,256 @@
+<?php
+/**
+ * WordPress_Sniffs_WP_I18nSniff
+ *
+ * Makes sure internationalization functions are used properly
+ *
+ * @category  PHP
+ * @package   PHP_CodeSniffer
+ * @author    Shady Sharaf <shady@x-team.com>
+ */
+class WordPress_Sniffs_WP_I18nSniff extends WordPress_Sniff {
+
+	public $i18n_functions = array(
```

Comment:
Wouldn't it be better to flip this array and have the function names as keys (this is standard in most other sniffs)? Then you can use `isset()` instead of `in_array()`. And perhaps more importantly, you could let the nature of each function be specified by the values, rather than [hardcoding this below](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/pull/136/files#diff-df68b36623d243f82931be87d069e1efR106). So you could have:

```php
array(
    '__' => 'simple',
    '_x' => 'context',
    '_n' => 'plural',
    '_nx' => 'plural_context',
)
```

ect.

Because right now if someone ever wanted to sub-class this and add their own custom translation functions (bad idea, I know), they'd have to duplicate all of the code rather than just add them to this list, because how each function is handled has to be hard-coded below.

Anyway, I just prefer `isset()` to `in_array()`. :smile: